stakingcontract/sc_utils.go

```diff
@@ -0,0 +1,51 @@
+package sc
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/Evrynetlabs/evrynet-node"
+	"github.com/Evrynetlabs/evrynet-node/common"
+	"github.com/Evrynetlabs/evrynet-node/core/types"
+	"github.com/Evrynetlabs/evrynet-node/evrclient"
+)
+
+// CheckTransStatus returns stratus of a transaction
+func CheckTransStatus(client *evrclient.Client, tx *types.Transaction) error {
```

Comment:
Utility function should in lib, since checking the status of a tx is not limited to the scope of staking contract.

Reply:
i moved it to lib/txutil